src/components/SharedComponents/Map/Map.tsx

```diff
@@ -294,7 +314,25 @@ const Map = ( {
     }
   };
 
-  const mapRegion = setRegion( );
+  const overlayObsTiles = ( ( withPressableObsTiles || withObsTiles )
+                            && ( Platform.OS !== "android"
+                              || previousTileUrl === tileUrlTemplate ) );
+
+  // In Android, when we render a single frame of <MapView> without <UrlTile>
+  // we use a tiny region increase of 0.001% to get onRegionChangeComplete to
+  // fire and update the obs tile url. This change is too small to be visible.
+  const fuzzRegion = curRegion => ( ( Platform.OS === "android"
+        && curRegion !== null && previousTileUrl !== tileUrlTemplate )
+    ? {
+      ...curRegion,
+      latitudeDelta: 1.00001 * curRegion.latitudeDelta,
+      longitudeDelta: 1.00001 * curRegion.longitudeDelta
+    }
+    : curRegion );
+  const mapRegion = fuzzRegion( setRegion( ) );
+  const mapInitialRegion = mapRegion === null
+    ? fuzzRegion( initialRegion )
+    : initialRegion;
```

Comment:
I find this hard to read. If you have a function called `fuzzRegion()` I expect it to fuzz a region, not fuzz a region sometimes. I also don't understand why you're fuzzing a region for `mapRegion`, then potentially fuzzing it again if somehow it was null. If this has to do with different behavior in iOS and Andoid, please be more clear in your variable and method naming, or have some clear branching logic for iOS and Android. I understand from your comment *why* you're doing this, and it does work, but I think the code could express how it works more clearly.

`setRegion()` is also confusing, even though that's not your problem, per se. It doesn't seem to actually set a region, just get an initial region for... reasons. @albullington do you remember why you named it that way? It seems more like an accessor than a mutator, but it's accessing a "region" that is not the same as the `region` prop.

Reply:
No idea, I'm pretty sure like many things this was a refactor I did of something that was named long ago.

Reply:
Thanks, @joergmlpts, I find this version easier to read. The only additional change I might make is to pull `fuzzRegion()` out of the component. It doesn't need to be redefined on every render.